Sources/SuperwallKit/Paywall Presentation/Internal Presentation/Presentation State/PaywallState.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
+//  
+//
+//  Created by Yusuf Tör on 07/09/2022.
+//
+
+import Foundation
+
+/// The current state of a paywall.
+public enum PaywallState {
```

Comment:
Does this need to be `public`

Reply:
Yeah this is what's returned from the track function

Reply:
Do we need a objc one?

Reply:
Nope